main.py

```diff
@@ -231,10 +231,10 @@ def ui_access_wrapper(self, *args, **kwargs):
         admin.AdminTopicsCsvFileDownloader),
     get_redirect_route(
         r'/addcontributionrightshandler',
-        admin.AddContributionRightsHandler),
+        admin.ContributionRightsHandler),
     get_redirect_route(
         r'/removecontributionrightshandler',
-        admin.RemoveContributionRightsHandler),
+        admin.ContributionRightsHandler),
```

Comment:
Please have one URL `/contributionrightshandler` for one handler*

Reply:
Done